### ISSUE TITLE
Support '\' path convention

### ISSUE THREAD

**Issue by techwiz24** [Dman95/SASM#45]
OS: Windows 7 x64
Configuration: MASM (6.15)

I'm trying to get SASM setup for use with masm 615 for one of my classes. For now, I'm using these options:

* Assembler Options: `/nologo /Sn /Sa /c /Fo$PROGRAM.OBJ$ $SOURCE$d`
* Linker Options: `/NOLOGO "$PROGRAM.OBJ$","$PROGRAM$",,,,

However, when passing `$PROGRAM$` and `$PROGRAM.OBJ$` as arguments to the assembler or linker, it seems to pass the paths with `/` instead of `\` (Ex: `c:/path/to/my/program.asm` instead of `c:\path\to\my\program.asm`). That version of masm that I'm trying to use seems to confuse that with option flags:

```
[17:29:36] Warning! Errors have occurred in the build:
LINK : warning L4017: /Users : unrecognized option name; option ignored
LINK : warning L4017: /nathan : unrecognized option name; option ignored
LINK : warning L4017: /AppData : unrecognized option name; option ignored
LINK : warning L4017: /Local : unrecognized option name; option ignored
LINK : warning L4017: /Temp : unrecognized option name; option ignored
LINK : warning L4017: /SASM : unrecognized option name; option ignored
LINK : warning L4017: /program.o : unrecognized option name; option ignored
LINK : warning L4017: /Users : unrecognized option name; option ignored
LINK : warning L4017: /nathan : unrecognized option name; option ignored
LINK : warning L4017: /AppData : unrecognized option name; option ignored
LINK : warning L4017: /Local : unrecognized option name; option ignored
LINK : warning L4017: /Temp : unrecognized option name; option ignored
LINK : warning L4017: /SASM : unrecognized option name; option ignored
LINK : warning L4017: /SASMprog.exe : unrecognized option name; option ignored
LINK : fatal error L1093: C:.obj : object file not found
```

I have tried quoting the path in both single and double quotes, and it seems to only like `\` instead of `/`.

Please provide an option (at least for masm) to format the path that the variable is replaced with to use `\` instead of `/`..

**Comment by techwiz24** [comment]
I need to do some testing, but I think patching:

```cpp
QString Common::pathInTemp(QString path)
{
    QString temp = QDir::tempPath();
    QChar lastSymbol = temp[temp.length() - 1];
    if (lastSymbol == QChar('/') || lastSymbol == QChar('\\')) {
        temp.chop(1);
    }
    if (! QFile::exists(temp + "/SASM")) {
        QDir().mkpath(temp + "/SASM");
    }

    return temp + "/SASM/" + path;
}
```

to return instead:

```cpp
return QDir::toNativeSeparators(temp + "/SASM/" + path);
```

might fix it.

**Comment by Dman95** [comment]
Yes. It seems working. I'll fix, test and build new version ASAP.

**Comment by Dman95** [comment]
I have fixed problem. You can see fix in 3.3 version.
